### PR TITLE
fix headers in PhysicsTools/PatAlgos

### PR DESCRIPTION
**Pull request by mrodozov**
Fix headers related to #15248

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: PhysicsTools/PatAlgos/plugins/BaseMVAValueMapProducer.h [hunk 1]
```diff
@@ -40,6 +40,9 @@
 #include "CommonTools/Utils/interface/StringObjectFunction.h"
 #include "DataFormats/Common/interface/ValueMap.h"
 #include "CommonTools/Utils/interface/TMVAZipReader.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
```
Comment (perrotta) [inline PhysicsTools/PatAlgos/plugins/BaseMVAValueMapProducer.h]: Can't these three includes get now removed from PhysicsTools/PatAlgos/plugins/BaseMVAValueMapProducer.cc, then?

**Comment by perrotta** [comment]
@mrodozov : please let us know if you plan to clean up the implementation file as suggested in https://github.com/cms-sw/cmssw/pull/23370#pullrequestreview-124279147, or if you prefere to let things as they are now instead

**Comment by mrodozov** [comment]
please test

**Comment by mrodozov** [comment]
@perrotta updated

**Comment by perrotta** [comment]
Thank you Mircho!
Then, I won't be shy and I'll ask the same also for #23463 (as well as  
from previous #23385)


Mircho Rodozov <notifications@github.com> ha scritto:

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**